ci-operator/config/openshift/external-dns/openshift-external-dns-master.yaml

```diff
@@ -1,34 +1,20 @@
-base_images:
-  base:
-    name: "4.12"
-    namespace: ocp
-    tag: base
-  ocp_builder_rhel-8-golang-1.18-openshift-4.12:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-1.18-openshift-4.12
 build_root:
   from_repository: true
 images:
 - dockerfile_path: Dockerfile.openshift
-  from: base
-  inputs:
-    ocp_builder_rhel-8-golang-1.18-openshift-4.12:
-      as:
-      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12
   to: external-dns
 promotion:
   name: external-dns
   namespace: external-dns-operator
 releases:
   initial:
     integration:
-      name: "4.12"
+      name: "4.13"
```

Comment:
Shouldn't this name be the version number of external-dns, not openshift?

Reply:
No, this is the version of OCP cluster where the tests will be run.